index.html

```diff
@@ -7,42 +7,108 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .header {
-      text-align: center;
+      text-align: center;                 
+      font-size: 24px;
+      font-weight: bold;
+      color: #2f4f4f;
+            }
     }
   </style>
+    <style>
+    .mainparagraph {
+      text-align: center;                 
+      font-size: 20px;
+      color: black;
+            }
+    }
+  </style>
+    <style>
+    .auxparagraph {
+      text-align: right;                 
+      font-size: 10px;
+      font-style: oblique;
+      color: #333;
+            }
+    }
+  </style>
+   
+
+   <style>
+  table, th, td {
+   border: 1px solid grey;
+   height: 25px;
+   vertical-align: bottom;
+   text-align: left;
+   color: C0C0C0;
+   font-size: 15px;
+
+ }
+   tr:hover {
+   background-color:#333}
```

Comment:
I like the idea of a hover effect, although this is so dark it makes the text unreadable.